From d4ba16770a74ca692df86a0356a938741e4e9e47 Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Thu, 3 Jan 2019 11:07:24 +0530
Subject: [PATCH 034/235] arm64: dts: ls1012a: remove unused gemac-bus-id

gemac-bus-id is unused property and is removed.

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
[Xulin: Original patch taken from NXP LSDK-19.06.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1012a-2g5rdb.dts | 2 --
 arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts   | 2 --
 arch/arm64/boot/dts/freescale/fsl-ls1012a-frwy.dts   | 1 -
 arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts    | 2 --
 arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts    | 2 --
 5 files changed, 9 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a-2g5rdb.dts b/arch/arm64/boot/dts/freescale/fsl-ls1012a-2g5rdb.dts
index 22e737993ce9..49713b447994 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-2g5rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-2g5rdb.dts
@@ -94,7 +94,6 @@
 		#address-cells = <1>;
 		#size-cells = <0>;
 		reg = <0x0>;	/* GEM_ID */
-		fsl,gemac-bus-id = <0x0>;	/* BUS_ID */
 		fsl,mdio-mux-val = <0x0>;
 		phy-mode = "sgmii-2500";
 		phy-handle = <&sgmii_phy1>;
@@ -105,7 +104,6 @@
 		#address-cells = <1>;
 		#size-cells = <0>;
 		reg = <0x1>;	/* GEM_ID */
-		fsl,gemac-bus-id = < 0x0>;	/* BUS_ID */
 		fsl,mdio-mux-val = <0x0>;
 		phy-mode = "sgmii-2500";
 		phy-handle = <&sgmii_phy2>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts b/arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts
index f658b7b71008..b6cdac49ffec 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts
@@ -89,7 +89,6 @@
 		#address-cells = <1>;
 		#size-cells = <0>;
 		reg = <0x0>;	/* GEM_ID */
-		fsl,gemac-bus-id = <0x0>;	/* BUS_ID */
 		fsl,mdio-mux-val = <0x0>;
 		phy-mode = "sgmii";
 		phy-handle = <&sgmii_phy1>;
@@ -100,7 +99,6 @@
 		#address-cells = <1>;
 		#size-cells = <0>;
 		reg = <0x1>;	/* GEM_ID */
-		fsl,gemac-bus-id = <0x1>;	/* BUS_ID */
 		fsl,mdio-mux-val = <0x0>;
 		phy-mode = "sgmii";
 		phy-handle = <&sgmii_phy2>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a-frwy.dts b/arch/arm64/boot/dts/freescale/fsl-ls1012a-frwy.dts
index bfe4ad5da76d..c6201bab8939 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-frwy.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-frwy.dts
@@ -155,7 +155,6 @@
 		#address-cells = <1>;
 		#size-cells = <0>;
 		reg = <0x1>;	/* GEM_ID */
-		fsl,gemac-bus-id = <0x1>;	/* BUS_ID */
 		fsl,mdio-mux-val = <0x0>;
 		phy-mode = "sgmii";
 		phy-handle = <&sgmii_phy2>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts b/arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts
index d716b86de563..382df05ff813 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts
@@ -147,7 +147,6 @@
 		#address-cells = <1>;
 		#size-cells = <0>;
 		reg = <0x0>;	/* GEM_ID */
-		fsl,gemac-bus-id = <0x0>;	/* BUS_ID */
 		fsl,mdio-mux-val = <0x2>;
 		phy-mode = "sgmii-2500";
 		phy-handle = <&sgmii_phy1>;
@@ -158,7 +157,6 @@
 		#address-cells = <1>;
 		#size-cells = <0>;
 		reg = <0x1>;	/* GEM_ID */
-		fsl,gemac-bus-id = <0x1>;	/* BUS_ID */
 		fsl,mdio-mux-val = <0x3>;
 		phy-mode = "sgmii-2500";
 		phy-handle = <&sgmii_phy2>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts
index 627a54520d42..d7f13bc976c7 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts
@@ -58,7 +58,6 @@
 		#address-cells = <1>;
 		#size-cells = <0>;
 		reg = <0x0>;	/* GEM_ID */
-		fsl,gemac-bus-id = <0x0>;	/* BUS_ID */
 		fsl,mdio-mux-val = <0x0>;
 		phy-mode = "sgmii";
 		phy-handle = <&sgmii_phy>;
@@ -69,7 +68,6 @@
 		#address-cells = <1>;
 		#size-cells = <0>;
 		reg = <0x1>;	/* GEM_ID */
-		fsl,gemac-bus-id = < 0x1 >;	/* BUS_ID */
 		fsl,mdio-mux-val = <0x0>;
 		phy-mode = "rgmii-txid";
 		phy-handle = <&rgmii_phy>;
-- 
2.17.1

